test.py

```diff
@@ -54,3 +54,17 @@ def test_units(registry, unit_str, dim, new_def):
 
     # Units can be parsed and have the correct dimensionality
     assert registry('1 ' + unit_str).dimensionality == dim
+
+
+@pytest.mark.parametrize('context, value',
+                         [('AR5GWP100', 28),
+                          ('AR4GWP100', 25),
+                          ('SARGWP100', 21)])
+def test_units_emissions(registry, context, value):
+    # The registry shouldn't convert with specifying a valid context
+    with pytest.raises(pint.DimensionalityError):
+        registry['ch4'].to('co2')
+
+    context = f'gwp_{context}'
+    assert registry['ch4'].to('co2', context).magnitude == value
+    assert registry['Mt ch4'].to('Mt co2', context).magnitude == value
```

Comment:
One question here - is it worth also testing `CO2-eq` in addition to `co2`? (only because this is the nominal unit used in reported Kyoto gas baskets - but maybe I'm being too fearful =P)

Reply:
The way it is implemented in this PR (see [here](https://github.com/danielhuppmann/units/blob/d6d4c8ace0b39c85dd987d9280298f760526e402/modules/emissions/emissions.txt#L8)) is that `['CO2-equivalent', 'CO2-eq', 'CO2eq', 'CO2e']` (and also same with `co2`) are just aliases for `co2`, rather than having an additional base unit. After all, they are equivalent...

Added a second round of tests anyway.

Reply:
Thank you!